platform/firecamp-platform/src/components/containers/SidebarContainer.tsx

```diff
@@ -118,8 +118,8 @@ const actionBarItems = [
         tabIndex={-1}
       />
     ),
-    text: `Settings (${scPrefix} ⇧ /)`,
     item: EActivityBarItems.Settings,
+    toolTip: `Settings (${scPrefix} ⇧ /)`,
```

Comment:
please change the key name to `tooltip` in the whole file.

Reply:
done